README.md

```diff
@@ -79,23 +79,35 @@ class ExampleEvaluator:
     submission_file_path = client_payload["submission_file_path"]
     aicrowd_submission_id = client_payload["aicrowd_submission_id"]
     aicrowd_participant_uid = client_payload["aicrowd_participant_id"]
-
+    
     submission = pd.read_csv(submission_file_path)
     # Or your preferred way to read your submission
 
     """
     Do something with your submitted file to come up
     with a score and a secondary score.
-    if you want to report back an error to the user,
+
+    If you want to report back an error to the user,
     then you can simply do :
       `raise Exception("YOUR-CUSTOM-ERROR")`
+
      You are encouraged to add as many validations as possible
      to provide meaningful feedback to your users
     """
     _result_object = {
         "score": np.random.random(),
         "score_secondary" : np.random.random()
     }
+    
+    media_dir = '/tmp/'
+
+    """
+    To add media to the result object such that it shows on the challenge leaderboard:
+    - Simply save the file at '/tmp/<filename>'
+    - Add the path of the media to the result object:
+        For images, add file path to _result_object["media_image_path"]
+        For videos, add file path to _result_object["media_video_path"]
+    """
```

Comment:
Maybe save an image to `/tmp/filename` and add it to the `_result_object` as intended.

Reply:
any photo would do

Reply:
And also about thumbnail images,  and how to add them

Reply:
Example things that you can do here
```python
   [1]:from PIL import Image, ImageDraw 
   ...:   
   ...: img = Image.new('RGB', (300, 30), color = (73, 109, 137)) 
   ...:   
   ...: d = ImageDraw.Draw(img) 
   ...: d.text((10,10), "AIcrowd Example Leaderboard Image :)", fill=(255,255,0)) 
   ...:   
   ...: img.show()                                                                                                                                 

```

Reply:
basically add an example right?